backend/core/types/src/backend-swagger.ts

```diff
@@ -798,17 +798,17 @@ export class UserService {
     })
   }
   /**
-   * Verifies token is valid
+   * Resend confirmation
```

Comment:
Just a note on dealing with this file. You may have done this, often I find that git does a terrible job trying to reconcile the differences, so I'll just get in a state where the backend can boot with it and then run generate:client so it's accurate.